graylog2-web-interface/src/components/alertconditions/AlertCondition.jsx

```diff
@@ -28,15 +28,25 @@ const AlertCondition = React.createClass({
   },
   _onUpdate(event) {
     event.preventDefault();
-    const request = {
-      type: this.props.alertCondition.type,
-      parameters: this.refs.updateForm.getValue(),
-    };
+    const request = this.refs.updateForm.getValue();
+    request.type = this.props.alertCondition.type;
     AlertConditionsActions.update.triggerPromise(this.props.alertCondition.stream_id, this.props.alertCondition.id, request)
     .then(() => {
       this.setState({edit: false});
     });
   },
+  _formatTitle() {
+    const alertCondition = this.props.alertCondition;
+    const alertConditionType = new AlertConditionsFactory().get(alertCondition.type);
+    const title = alertCondition.title ? alertCondition.title : `Untitled ${alertConditionType.title} condition`;
```

Comment:
I would simply leave out the title if not present, I don't see the need of adding that "Untitled" prefix. I think specially at the beginning will look really weird, as none of the conditions will have title, and all will contain the untitled prefix.